documentation/docs/developers/strategies/twap.md

```diff
@@ -0,0 +1,53 @@
+# TWAP
+
+Unlike the previous tutorial strategies which demonstrated straight-forward tasks like adding configurations and using clock ticks to periodically perform action, this strategy will introduce you to implement a more complicated algorithm. The tools demonstrated in this strategy is : 
+1) Maintaining some states over clock ticks to retain important information that will decide algorithm's next action 
+2) Quantizing order size depending of user's specification of how the user wants order to be executed.
+
+## Overview
+
+TWAP strategy provides the guide for you to add these features : 
+
+* Divide up an order into *n* individual orders
+* Add time delay between each individual orders
+
+Note that unlike Simple Trade strategy which places one order of entire quantity, the number of order is dynamically determined by user's input. Since one order can be placed over a single clock tick, there needs to some sort of information about order 'state' that will indicate when the order should stop.
```

Comment:
The first sentence can be simplified. It can very well just start with "Unlike the simple trade strategy ... "

The meaning of the second sentence is unclear. You want to stress the fact that because only one order is placed in a clock tick, a state machine is needed to emit multiple orders over different clock ticks. Need to improve the wording a bit.